File: Models/Menu.cs
using System.Collections.Generic;

namespace React.Models
{
    public class Menu
    {
        public string RestaurantName { get; set; }
        public IEnumerable<Item> Items { get; set; }

        public Menu()
        {
            Items = new List<Item>();
        }
    }
}

File: Assets/js/components/Menu.js
import React from 'react';
import MenuItemContainer from '../containers/MenuItemContainer';

const Menu = ({ menu }) => (
    <div className="c-listing">
        <h1>{menu.restaurantName}</h1>

        {menu.items.map(item => <MenuItemContainer item={item} key={item.id} />)}
    </div>
);

export default Menu;
